src/utils.jl

```diff
@@ -75,6 +76,7 @@ function src_from_file_or_REPL(origin::AbstractString, args...)
     if m !== nothing
         return src_from_REPL(m.captures[1], args...)
     end
+    isfile(origin) || return ""
```

Comment:
Would it be better to return `nothing` here to indicate a failure and have the caller deal with it?